packages/viewer/src/modules/SceneObjects.js

```diff
@@ -295,7 +295,7 @@ export default class SceneObjects {
 
         let materialId = `${m.type}/${m.vertexColors}/${m.color.toJSON()}/${m.side}/${
           m.transparent
-        }/${m.opactiy}/${m.emissive}/${m.metalness}/${m.roughness}/${m.wireframe}`
+        }/${m.opacity}/${m.emissive}/${m.metalness}/${m.roughness}/${m.wireframe}`
```

Comment:
Please review closely

Reply:
It's a fun catch. I think it should be fine as these ids are transient/local session only relevant, but I'll ping @cristi8 to double check as it was his brain at work here :)

Reply:
oh, yes, nice catch. this is actually a bug.. we would now group meshes with materials that only differ in opacity value, but have the same other attributes -> resulting in wrong materials for some objects in the final rendered scene